src/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required (VERSION 3.5)
 
+include (agentRules)
+
+compileasc99 ()
```

Comment:
Important Note: we don't need to ever state this again. As long as the head file is stating which standard to use all child targets will inherit it. 

The only places we need to specify compileasc99() is in new projects (not sure if that effects existing code within the POC)

Reply:
we also do not need to state `cmake_minimum_required (VERSION 3.5)` anywhere else either, I'm pretty sure.